test/unit/service_test.rb

```diff
@@ -569,8 +569,7 @@ class AsynchronousDeletionOfService < ActiveSupport::TestCase
     service = Service.new(account: account = Account.new)
     service.proxy = Proxy.new(service: service)
 
-    account.stubs(:provider_can_use?).with(:apicast_v1).returns(true) # irrelevant for this test
-    account.stubs(:provider_can_use?).with(:apicast_v2).returns(true) # irrelevant for this test
+    account.stubs(:provider_can_use?).with(:apicast_v2).returns(true)
```

Comment:
We should keep the comment in the end of the line 
```suggestion
    account.stubs(:provider_can_use?).with(:apicast_v2).returns(true) # irrelevant for this test
```